responders/src/UI/embc-responder/src/app/feature-components/wizard/profile-components/profile-review/profile-review.component.ts

```diff
@@ -61,13 +54,48 @@ export class ProfileReviewComponent implements OnInit {
   }
 
   /**
-   * Updates the tab status, step status and navigates
-   * to the next step
+   * Submit evacuee profile and continue to Step 2
    */
   save(): void {
+    this.stepCreateProfileService.nextTabUpdate.next();
+
     if (this.verifiedProfileFC.valid) {
-      this.stepCreateProfileService.setTabStatus('review', 'complete');
-      this.wizardService.setStepStatus('/ess-wizard/create-ess-file', false);
+      this.saveLoader = true;
+
+      this.evacueeProfileService
+        .upsertProfile(this.stepCreateProfileService.createProfileDTO())
+        .subscribe(
+          (profileId) => {
+            this.cacheService.set('profileId', profileId);
```

Comment:
Use a service to set and get profileId

Reply:
I'll add get/set functions for this to **evacuee-profile.service**.

**evacuee-search-results.component**, **wizard.component**, and **person-detail-form.component** are all directly interacting with the cache service as well.

Reply:
@MShayOffice  If you are looking for code references try LoadTeamListService or LocationsService not the components that are presently under development

Reply:
@SodhiA1 Not looking for references, just pointing out that there are other places where this type of thing has been merged in.